### DISCUSSION TITLE
Memory

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#118] [Q&A] [answered]
I have 512gb storage ssd but on home it has a lot of memory but only 15 gb for operating system is being shown. Is this an issue if yes please help. Please give some insights as I am new to all this.

**Comment by T-Crypt** [comment]
@GeekWolf007 

This is because during the "archinstall" process  you selected for a separate /home partition on your main storage device.

**Reply by sudobrat** [reply]
Can I change that now ? I have used 12 gb out of it to be safe I want to add some storage just to be sure

**Reply by T-Crypt** [reply, marked as answer]
@GeekWolf007 

Yes, just download gnome-disk-utility via pacman or yay 

yay -S gnome-disk-utility 

Once you have downloaded the disk utility you should be able to extend / expand the /HOME partition using free space on your main /dev/nvme or sda drive

**Reply by sudobrat** [reply]
Ok thanks 💝
